types/react-window/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for react-window 1.5
+// Type definitions for react-window 1.6
```

Comment:
You should merge the latest changes from `master`. This has already been bumped to 1.7.